benchmark/selectors/selector.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const fs = require("node:fs");
+const path = require("node:path");
+const process = require("node:process");
+const suite = require("../document-suite");
+const { JSDOM } = require("../..");
+
+exports.querySelectorAll = function () {
+  let document, selectors, total;
+  let count = 0;
+  let cycle = 0;
+
+  return suite({
+    setup() {
+      const dir = "./benchmark/selectors/sizzle/speed/";
+      const htmlFile = path.resolve(process.cwd(), dir, "./data/selector.html");
```

Comment:
Resolving relative to `process.cwd()` is a bad idea, since it prevents people from running the test in different places. (E.g., I think this will fail if one does `cd benchmark; node ./runner`.) Instead, use `__dirname`, since that never changes.

Reply:
Done.